packages/ubuntu_desktop_installer/lib/pages/allocate_disk_space/allocate_disk_space_dialogs.dart

```diff
@@ -191,7 +191,7 @@ class _RadioValueTile<T> extends StatelessWidget {
     return ValueListenableBuilder<T>(
       valueListenable: groupValue,
       builder: (context, groupValue, child) {
-        return RadioListTile<T>(
+        return RadioButton<T>(
```

Comment:
Vertical alignment of the two columns is broken, as evidenced by your "after" screenshot (and tested locally).

Reply:
Good catch, thanks!